app/views/fragments/checkout/fieldsPayment.scala.html

```diff
@@ -0,0 +1,32 @@
+@(form: Form[model.SubscriptionData])
+
+<div class="form-field js-checkout-account">
+    <label class="label" for="payment-account">Bank account number</label>
+    <input type="text" pattern="[0-9]*" minlength="6" maxlength="10" class="input-text js-input" id="payment-account" name="payment.account">
+    @fragments.forms.errorMessage("Please enter between 6 and 10 digits for the account number")
+</div>
+<div class="form-field js-checkout-sortcode">
+    <label class="label" for="payment-sortcode">Bank sort code</label>
+    <input type="text" class="input-text input-text--small js-input"
+        name="payment.sortcode" id="payment-sortcode" maxlength="8"
+        placeholder="00-00-00" required
+        data-masker-delim="-" data-masker-length="2">
+    @fragments.forms.errorMessage("")
+</div>
+<div class="form-field js-checkout-holder">
+    <label class="label" for="payment-holder">Name of account holder</label>
+    <input type="text" maxlength="18" class="input-text js-input" id="payment-holder" name="payment.holder">
+    @* TODO: Why limited to 18 characters? *@
```

Comment:
Apparently it's part of the BACS service limits http://www.bacs.co.uk/Bacs/Businesses/Resources/Pages/Glossary.aspx

"The payer’s account name (maximum of 18 characters). This must be the name of the person who is paying the Direct Debit and has signed the Direct Debit Instruction (DDI)."